<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushLogin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushClear">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePassword">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUser">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushContact">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact Us</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCreate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushContact_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Supported</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[+]Status</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Testimonals</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Our Guarantee</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sales Pitch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Concept of Operations</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Services</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Help</string>
     </property>
    </item>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>pushLogin</zorder>
   <zorder>pushClear</zorder>
   <zorder>linePassword</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushContact</zorder>
   <zorder>buttonCreate</zorder>
   <zorder>Logo</zorder>
   <zorder>pushContact_2</zorder>
   <zorder>Status</zorder>
   <zorder>lineUser</zorder>
   <zorder>comboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushClear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushContact</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonContact()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushLogin</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
  <slot>buttonContact()</slot>
  <slot>buttonLogin()</slot>
 </slots>
</ui>
